libhkcnb-nodejs/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "libhkcnb-nodejs"
```

Comment:
I've already changed the name on this three times, so I'm kind of out of other ideas. If you have an idea here, let me know. 

No, this is unlikely to be published to crates.io at this point, but it's possible we may want to since `github.com/heroku/heroku-buildpacks-nodejs` could use the `resolve_version` binary included within.

Reply:
I guess my main concerns were:
- that someone searching creates io for "libcnb" would get a crate that doesn't have anything to do with libcnb.rs
- that we're overusing the "lib*" thing (which isn't really a naming prefix used in the Crates ecosystem)

Re resolve_version:
- `cargo install` supports git paths, so the classic buildpack could always point at this git repo for the transition period (of transitioning away from classic buildpacks) - would mean one less step for having to release new versions (of not needing to publish to crates.io). See: https://doc.rust-lang.org/cargo/commands/cargo-install.html
- if it was wanted as a crate, it almost seems like it might be best as it's own crate, rather than in a generic "nodejs utils" crate anyway?

Anyway, seems fine to leave as is for now though, can always revisit later prior to publishing, if that happens.

Reply:
I've moved it to `common/nodejs-utils` and called it `heroku-nodejs-utils`. I like it a little better, and it's not really affiliated with any other project.